views/sections/forms/enroll.ejs

```diff
@@ -0,0 +1,67 @@
+
+                        <!-- First Name -->
+                        <div class="input-group col-lg-6 mb-4">
+                            <div class="input-group-prepend">
+                            <span class="px-1 border-md border-right-0 me-2">
+                                <img class="" src="/public/icon/profile.png">
+                            </span>
+                            </div>
+                            <input id="firstName" type="text" name="firstname" placeholder="İsim" class="form-control bg-white border-left-0 border-md">
+                        </div>
+
+                        <!-- Last Name -->
+                        <div class="input-group col-lg-6 mb-4">
+                            <div class="input-group-prepend">
+                            <span class="px-1 border-md border-right-0 me-2">
+                                <img class="" src="/public/icon/profile.png">
+                            </span>
+                            </div>
+                            <input id="lastName" type="text" name="lastname" placeholder="Soyisim" class="form-control bg-white border-left-0 border-md">
+                        </div>
+
+                        <!-- Email Address -->
+                        <div class="input-group col-lg-12 mb-4">
+                            <div class="input-group-prepend">
+                            <span class="px-1 border-md border-right-0 me-2">
+                                <img class="" src="/public/icon/mail.png">
+                            </span>
+                            </div>
+                            <input id="email" type="email" name="email" placeholder="Email Address" class="form-control bg-white border-left-0 border-md">
+                        </div>
+
+                        <!-- Phone Number -->
+                        <div class="input-group col-lg-6 mb-4">
+                            <div class="input-group-prepend">
+                            <span class="px-1 border-md border-right-0 me-2">
+                                <img class="" src="/public/icon/phone.png">
+                            </span>
+                            </div>
+                            <input id="phoneNumber" type="tel" name="phone" placeholder="Phone Number" class="form-control bg-white border-md border-left-0">
+                        </div>
+
+                        <!-- Student Number -->
+                        <div class="input-group col-lg-12 mb-4">
+                            <div class="input-group-prepend">
+                            <span class="px-1 border-md border-right-0 me-2">
+                                <img class="" src="/public/icon/student-number.png">
+                            </span>
+                            </div>
+                            <input id="email" type="text" name="Student Number" placeholder="Öğrenci Numarası" class="form-control bg-white border-left-0 border-md">
+                        </div>
```

Comment:
You're missing some fields like grade, group chat options etc. Please see the [current form](https://formie.div72.xyz/forms/3?goto=https://ozguryazilimhacettepe.com/welcome).